packages/terra-application-utility/src/utility/_UtilityMenuItem.module.scss

```diff
@@ -145,4 +145,17 @@
   .menu-item-fill {
     width: 100%;
   }
+
+  .read-only {
+    cursor: text;
+
+    &:hover {
+      background-color: var(--terra-application-header-utility-menu-body-item-hover-background-color);
```

Comment:
we should sync with UX, but I think a read only item will never have hover and focus states. We'd probably set `background-color: transparent` and `box-shadow: none`, instead of opening them up for themes

Reply:
I thought so too, but wasn't sure if I should leave the option there just in case. Will take it out!

Reply:
I need to review PR further, but generally anything that is "read-only" or non-interactive elements on the page never should have have any visual state changes (pointer/color,background-color/shadow etc.) for hover, focus, active, etc. (for all themes)

Reply:
All the focus styles have been removed, the only remaining question I have is should we still allow people to focus to it via keyboard tab? I'm thinking we drop that functionality from read-only items in the menu as it should be non-interactive, e.g. no clicking on it to trigger something.

Reply:
I was originally thinking altering the tabindex would also hide the information from a screen reader, but I realized you can't tab to it anyway, so I threw the conditional back in there.